Server/Source/player_pool.hpp

```diff
@@ -1419,6 +1419,11 @@ struct PlayerPool final : public IPlayerPool, public NetworkEventHandler, public
 				return false;
 			}
 
+			if (std::fabs(1.0 - glm::length(unoccupiedSync.Roll)) >= 0.000001)
```

Comment:
worth turning this magic number into a const and documenting the rationale behind the precision